unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderEntity/DCLBuilderInWorldEntity.cs

```diff
@@ -109,21 +109,21 @@ public bool HasSmartItemComponent()
 
     public bool HasSmartItemActions()
     {
-        SmartItemComponent component = (SmartItemComponent)rootEntity.GetBaseComponent(CLASS_ID_COMPONENT.SMART_ITEM);
-
+        BaseComponent component;
+        rootEntity.TryGetBaseComponent(CLASS_ID_COMPONENT.SMART_ITEM, out component);
+     
         if (component == null)
             return false;
```

Comment:
```suggestion
	if(!rootEntity.TryGetBaseComponent(CLASS_ID_COMPONENT.SMART_ITEM, out BaseComponent component))     
            return false;
```

Reply:
the point of using the `TryGet` approach was being able to summarize these checks like this.